src/main/java/org/tron/core/db/Manager.java

```diff
@@ -207,6 +218,13 @@
   @Getter
   private ForkController forkController = ForkController.instance();
 
+  private ExecutorService executorService = Executors
+      .newFixedThreadPool(1, r -> new Thread(r, "PreValidateTransactionSign"));
+
+  public static Map<ByteArrayWrapper, AccountCapsule> accountCache = new HashMap<>();
+
+  public volatile boolean done = false;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make done a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1916&issues=AWgnKDa4rt4Kct_4Nf2C&open=AWgnKDa4rt4Kct_4Nf2C)

[ISSUE_KEY]: AWgnKDa4rt4Kct_4Nf2C